<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-3IHQCB1" ignored="1" name="Test(failed)" tests="2" failures="2" timestamp="09 Nov 2021 10:47:35 GMT" time="0.085" errors="0">
  <testcase name="test37" time="0.015" classname="org.cts.A">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.cts.A.test37(A.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test37 -->
  <testcase name="test35" time="0.0" classname="org.cts.A">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.cts.A.test35(A.java:10)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test35 -->
  <testcase name="test36" time="0.0" classname="org.cts.A">
    <ignored/>
  </testcase> <!-- test36 -->
</testsuite> <!-- Test(failed) -->
